extension/config/src/main/resources/conf/platform/configuration.properties

```diff
@@ -663,6 +663,16 @@ wiki.editPage.livingTime=${exo.wiki.editPage.livingTime:1800000}
 
 ###########################
 #
+# social
+#
+# To disable any Activity Type add : exo.activity-type.<activity-type-key>.enabled=false
+# by default all activity Types are activated.
+# The list of all activity types defined by:
+
+exo.social.activity.types= ${exo.social.activity.types:DEFAULT_ACTIVITY,SPACE_ACTIVITY,USER_ACTIVITIES_FOR_SPACE,exosocial:relationship,LINK_ACTIVITY,sharecontents:spaces,USER_PROFILE_ACTIVITY,DOC_ACTIVITY,files:spaces,sharefiles:spaces,contents:spaces,cs-calendar:spaces,TaskAdded,ks-forum:spaces,ks-answer:spaces,ks-poll:spaces,ks-wiki:spaces,USER_ACTIVITIES_FOR_RELATIONSHIP}
```

Comment:
What is the goal of this property and what is the impact for a developer who wants to add its own custom activity type ?